quality/postman/emodb_tests.postman_collection.json

```diff
@@ -21757,95 +21757,18809 @@
+																},
+																{
+																	"key": "tag",
+																	"value": "postman"
+																},
+																{
+																	"key": "debug",
+																	"value": "true"
+																}
+															]
+														},
+														"description": "\"Creates or replaces a piece of content in the data store.  Overwrites the old\\n\" +\n                    \" version of the content, if it exists.  Expects a literal JSON representation\\n\" +\n                    \" of the object.\""
+													},
+													"response": []
+												},
+												{
+													"name": "Retrieves the current version of a piece of content from the data store",
+													"event": [
+														{
+															"listen": "test",
+															"script": {
+																"exec": [
+																	"pm.test(\"Status code is 200\", function () {",
+																	"    pm.response.to.have.status(200);",
+																	"});",
+																	"",
+																	"var jsonData = pm.response.json();",
+																	"",
+																	"var responsePropertiesMainActual = Object.keys(jsonData);",
+																	"var responsePropertiesAvailabilityActual = Object.keys(jsonData.availability);",
+																	"var responsePropertiesTemplateActual = Object.keys(jsonData.template);",
+																	"",
+																	"var responsePropertiesMainExpected = [\"availability\",\"id\",\"name\",\"template\",\"test_field\",\"~table\",\"~version\",\"~lastMutateAt\",\"~deleted\",\"~lastUpdateAt\",\"~id\",\"client\",\"~signature\",\"type\",\"~firstUpdateAt\"];",
+																	"var responsePropertiesAvailabilityExpected = [\"facade\",\"placement\"];",
+																	"var responsePropertiesTemplateExpected = [\"test_field\"];",
+																	"",
+																	"pm.test(\"Response fields\", function () {",
+																	"    pm.expect(responsePropertiesMainActual).to.have.deep.members(responsePropertiesMainExpected);",
+																	"    pm.expect(responsePropertiesAvailabilityExpected).to.have.deep.members(responsePropertiesAvailabilityExpected);",
+																	"    pm.expect(responsePropertiesTemplateActual).to.have.deep.members(responsePropertiesTemplateExpected);",
+																	"});",
+																	"",
+																	"delete jsonData[\"~lastMutateAt\"];",
+																	"delete jsonData[\"~lastUpdateAt\"];",
+																	"delete jsonData[\"~signature\"];",
+																	"delete jsonData[\"~firstUpdateAt\"];",
+																	"",
+																	"var stringToReplace = '{\"availability\":{\"facade\":false,\"placement\":\"ugc_global:ugc\"},\"id\":1,\"name\":\"postman_table_name\",\"template\":{\"test_field\":\"test_value_of_field\"},\"test_field\":\"postman\",\"~table\":\"postman_table_name\",\"~version\":1,\"~deleted\":false,\"~id\":\"postman_document_id\",\"client\":\"iuliia_customer\",\"type\":\"create_table_test\"}';",
```

Comment:
here and below
what's the reason to create string and than replace?
couldn't we just create proper string? or create JSON object and stringify?